src/server/web/index.ts

```diff
@@ -358,6 +358,12 @@ router.get('/flush', async ctx => {
 	await ctx.render('flush');
 });
 
+// streamingに非WebSocketリクエストが来た場合にbase htmlをキャシュ付きで返すと、Proxy等でそのパスがキャッシュされておかしくなる
+router.get('/streaming', async ctx => {
+	ctx.status = 503;
```

Comment:
[426](https://developer.mozilla.org/ja/docs/Web/HTTP/Status/426) の方が良さそう。

```suggestion
	ctx.status = 426;
	ctx.set('Upgrade', 'websocket');
```

Reply:
（42x 番台の扱いを 5xx 寄りで解釈していいのかは分からないけど）

Reply:
これ、明らかにクライアントが非WebSocketをリクエストした場合だけではなく、
クライアントが正当にWebSocketでリクエストしているのに、サーバーかアプリ側の不具合で？WebSocketしてハンドル出来てない可能性が疑われるので、4xxでクライアント側の原因と言い切りたくないというのがあります。

Reply:
なるほど、となると原因を究明できるまでは 5xx 番台の方が適当そうですね。503 だと若干意味が変わってくるような気もするので 502 か 500 あたり？

Reply:
そもそも、Cloudflare経由のWebSocketで503返してもクライアントまで503戻ってこないみたい。
たぶんCloudflareでステータスコード見て挙動を変えてそうなので、ステータスコードによって挙動変わるかも。
とりあえず今は503でちゃんと動いてます。

Reply:
ちょっと検証して見るかも

Reply:
https://github.com/syuilo/misskey/issues/6718#issuecomment-701233858 でUpgradeがないリクエストが来てるから426でUpgradeしたい気もするけど、sec-websocket-versionとか一部ヘッダだけ中途半端にあるから400な気もするし、Cloudflareの挙動がわからないからあまり503から変えたくもない気もする。

Reply:
> となると原因を究明できるまでは 5xx 番台の方が適当そうですね。

にLGTMなのでステータスコードについてはこのままで